lib/models/files.ts

```diff
@@ -598,7 +598,11 @@ export async function ensureUserPathIsValidAndSecurelyAccessible(userId: string,
 
   const resolvedFullPath = `${resolve(fullPath)}/`;
 
-  if (!resolvedFullPath.startsWith(userRootPath)) {
+  // Normalize path separators for consistent comparison on Windows
+  const normalizedUserRootPath = userRootPath.replace(/\\/g, '/');
+  const normalizedResolvedFullPath = resolvedFullPath.replace(/\\/g, '/');
+
+  if (!normalizedResolvedFullPath.startsWith(normalizedUserRootPath)) {
```

Comment:
This makes me pause. We have `@std/path` for any path operations instead of a manual check/fix. I also haven't heard of problems here.

Is this because you're sending a windows path from the client into a linux path in the server, for example?

In any case, it's simple enough and I'll allow it after testing, but I'd  request you change it to use `replaceAll` with a string instead of a regular expression, simply for performance, even if it's barely noticeable.

Reply:
Yeah, I'm developing locally on Windows with Docker Desktop - this is the third time I've run into this issue so I thought I'd address it directly. Everything else seems to be fine!

Reply:
Hey, I changed this as part of [3b7447f6cea873f5919c24282270f7fe727a8026](https://github.com/bewcloud/bewcloud/pull/106/commits/3b7447f6cea873f5919c24282270f7fe727a8026#diff-6f4b04585a5ee9df2dd524b2cc17e27ce5460299ed0b5bad9414d07ebebb7a0e), I'm not sure why it's not showing up as a change on here?

Reply:
I see the change.